lib/controllers/issueish-detail-controller.js

```diff
@@ -130,13 +153,13 @@ export class BareIssueishDetailController extends React.Component {
     const fromPullRefspec =
       headRemote.getOwner() === repository.owner.login &&
       headRemote.getRepo() === repository.name &&
-      headPush.getShortRemoteRef() === `pull/${issueish.number}/head`;
+      headPush.getShortRemoteRef() === `pull/${pullRequest.number}/head`;
```

Comment:
Okay, yeah, and this is guaranteed to be defined because we gate on `this.state.typename` up above. 👍